### PR TITLE
[MIG 12.0] mig geoengine partner

### PR DESCRIPTION
**Pull request by EBII**

### CODE REVIEW

File: geoengine_partner/views/partner.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="geo_partner_view_form">
+        <field name="name">geo_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <notebook>
+                <page string="Geoengine Map">
+                    <field name="partner_point" widget="geo_edit_map"
```
Comment (yvaucher) [inline geoengine_partner/views/partner.xml:10]: What is the reason behind the renaming of `geo_point` to `partner_point` ?

File: geoengine_partner/hooks.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    """
+    This post-init-hook will update all partner already localize with geo point.
+    """
+    env = api.Environment(cr, SUPERUSER_ID, dict())
+    # import pdb;pdb.set_trace()
```
Comment (yvaucher) [inline geoengine_partner/hooks.py:12]: Reminder of pdb to remove

File: geoengine_partner/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{'name': 'Geospatial support of partners',
+ 'version': '12.0.1.0.0',
+ 'category': 'GeoBI',
+ 'author': "Camptocamp, Odoo Community Association (OCA)",
+ 'license': 'AGPL-3',
+ 'website': 'https://github.com/OCA/geospatial',
+ 'depends': [
+     'base',
+     'base_geoengine',
+     'base_geolocalize'
```
Comment (yvaucher) [inline geoengine_partner/__manifest__.py:12]: Why was this dependency added? There is already a module `geoengine_base_geolocalize`

**Review by yvaucher — CHANGES_REQUESTED** [review]
@EBII please see my comments

You might also want to fix your commit authoring, it seems to not be correctly linked to your github account. Probably the email parameter in git config.
Plus, could you squash your last commits.

**Comment by Tonow-c2c** [comment]
@EBII I think only change like https://github.com/OCA/geospatial/pull/250/commits/2c6277a24ba3833cd753912d5cf8e32dcb58ee5e
can be enough to migrate the module

**Comment by yvaucher** [comment]
friendly reminder @EBII

**Comment by yvaucher** [comment]
Superseeded by #274

**Pull request closed without merging**